fluent-plugin-azure-storage-append-blob.gemspec

```diff
@@ -4,12 +4,12 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 Gem::Specification.new do |spec|
   spec.name    = "fluent-plugin-azure-storage-append-blob"
   spec.version = "0.1.0"
```

Comment:
i think we need to rev this number to 0.1.1

Reply:
updated